Godnoken/DOM-II
# DOM 2 - LambdaSchool

## What does this PR do?
 DOM 2 I Project MVP
​
## Description of tasks completed
- Implemented 10 unique event listeners
- Implemented demonstration of how to stop propagation and the use of preventDefault()

## How can this be tested?
Clone the project.
Open up the project files in your favourite code editor.
Check index.js for event triggers.
Run index.html to test event triggers on the webpage.
​
## Relevant screenshot
![image](https://user-images.githubusercontent.com/17499594/67869155-53d3cb80-fb2d-11e9-8119-6d709df8ec95.png)
